utils/watchcat/files/initd_watchcat

```diff
@@ -17,91 +17,114 @@ append_string() {
 
 time_to_seconds() {
 	local time=$1
-	unset seconds
 
 	{ [ "$time" -ge 1 ] 2>/dev/null && seconds="$time"; } ||
 		{ [ "${time%s}" -ge 1 ] 2>/dev/null && seconds="${time%s}"; } ||
 		{ [ "${time%m}" -ge 1 ] 2>/dev/null && seconds=$((${time%m} * 60)); } ||
 		{ [ "${time%h}" -ge 1 ] 2>/dev/null && seconds=$((${time%h} * 3600)); } ||
 		{ [ "${time%d}" -ge 1 ] 2>/dev/null && seconds=$((${time%d} * 86400)); }
+
+	echo $seconds
+	unset seconds
+	unset time
 }
 
 load_watchcat() {
-	config_get period $1 period
-	config_get mode $1 mode
-	config_get pinghosts $1 pinghosts
-	config_get ping_period $1 ping_period
-	config_get no_ping_time $1 no_ping_time
-	config_get force_delay $1 force_delay
-	config_get ping_size $1 ping_size "standard"
-
-	local nopingtime_dflt="900"
-	local force_delay_default="60"
-
-	# Fix potential typo in mode (backward compatibility).
+	config_get period "$1" period
+	config_get mode "$1" mode
+	config_get pinghosts "$1" pinghosts
+	config_get pingperiod "$1" pingperiod
+	config_get forcedelay "$1" forcedelay
+	config_get pingsize "$1" pingsize
+	config_get interface "$1" interface
+	config_get mmifacename "$1" mmifacename
+	config_get unlockbands "$1" unlockbands
+
+	local forcedelay_default="0"
```

Comment:
positive you want 0 here?

Reply:
I can put it back to 60s if that makes more sense for most users.  I just thought I would disable the sysreq reboot by default since it has the potential to lead to data loss.

Reply:
its kind of a weird place to static out such a hardcoded setting... one solution might be to use ":-N" here... and have a global option in uci for this...

or something equivalent... which can be ignored if... ' ! -z 'custom_reboot_script'... blah blah blah... etc. etc. i.e. i'd expect a few pre-conditions or 'alt-action-methods'... or even 'per-interface/watch' action method logic to proceed this eventually...

Reply:
I refactored using `config_get`s  parameter for a default value, which let me remove a lot of the conditional checks later on in the file.